lib/bloc/lnurl/lnurl_bloc.dart

```diff
@@ -37,14 +38,19 @@ class LNUrlBloc with AsyncActionsHandler {
   }
 
   listenLNUrl() {
+    log.info('listenLNUrl');
     if (_lnUrlStreamController == null) {
       _lnUrlStreamController = StreamController.broadcast();
       final injector = ServiceInjector();
       Rx.merge([
         injector.nfc.receivedLnLinks(),
         injector.lightningLinks.linksNotifications,
         _lnurlInputController.stream,
-        injector.device.distinctClipboardStream,
+
+        // Process lightning-addresses in the clipboard only when the user manually 
+        // pastes them in and they show up in _lnurlInputController.stream.
+        injector.device.distinctClipboardStream
+            .takeWhile((l) => !isLightningAddress(l)),
```

Comment:
`takeWhile` would close the stream on first not matched event.
Instead I think we should use:

`injector.device.distinctClipboardStream.where(isLightningAddress)`